profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -114,55 +113,62 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
       $state = $node->workbench_moderation['current']->state;
       $translation_possible = in_array($state, $states, 'strict');
 
-      $languages = tmgmt_available_languages();
-      $options = &$form['languages']['#options'];
+      $options = $form['languages']['#options'];
+
+      $existing_jobs = tmgmt_job_item_load_latest(
+        'workbench_moderation',
+        $entity_type,
+        $node->nid,
+        $node->translations->original
+      );
 
-      $existing_jobs = tmgmt_job_item_load_latest('workbench_moderation',
-        $entity_type, $node->nid, $node->translations->original);
       $always_show_direct_translation_links = variable_get('tmgmt_workbench_moderation_always_show_direct_translation_links', FALSE);
 
-      // Disable checkbox for languages already translated.
-      foreach ($languages as $langcode => $language) {
-        if (isset($node->translations->data[$langcode]) && !empty($existing_jobs)) {
-          $form['languages'][$langcode] = array(
+      foreach ($options as $lang_code => $option) {
```

Comment:
Then, instead of `foreach ($options as $lang_code => $option)`;  the use of an $lang_codes = array_keys($options) and loop on $lang_codes would be clearer?